apps/ui/src/components/TransactionsListItem.vue

```diff
@@ -1,27 +1,37 @@
 <script setup lang="ts">
 import { formatUnits } from '@ethersproject/units';
 import { _n, shorten } from '@/helpers/utils';
+import { getNames } from '@/helpers/stamp';
 import { Transaction } from '@/types';
 
-defineProps<{ tx: Transaction }>();
+const props = defineProps<{ tx: Transaction }>();
 
-function getTitle(tx: Transaction) {
-  if (tx._type === 'sendToken') {
-    return `Send <b>${_n(formatUnits(tx._form.amount, tx._form.token.decimals))}</b> ${
-      tx._form.token.symbol
-    } to <b>${shorten(tx._form.recipient)}</b>`;
-  }
+const title = ref<string>('');
 
-  if (tx._type === 'sendNft') {
-    return `Send <b>${_n(formatUnits(tx._form.amount, 0))}</b> NFT to <b>${shorten(
-      tx._form.recipient
-    )}</b>`;
-  }
+if (props.tx._type === 'sendToken') {
+  title.value = `Send <b>${_n(formatUnits(props.tx._form.amount, props.tx._form.token.decimals))}</b> ${
+    props.tx._form.token.symbol
+  } to <b>_NAME_</b>`;
+}
+
+if (props.tx._type === 'sendNft') {
+  title.value = `Send <b>${_n(formatUnits(props.tx._form.amount, 0))}</b> NFT to <b>_NAME_</b>`;
+}
 
-  if (tx._type === 'contractCall') {
-    return `Contract call to <b>${shorten(tx._form.recipient)}</b>`;
-  }
+if (props.tx._type === 'contractCall') {
+  title.value = `Contract call to <b>_NAME_</b>`;
 }
+
+const getParsedTitle = computedAsync(
```

Comment:
```suggestion
const parsedTitle = computedAsync(
```

It's not a function.

Reply:
This got resolved on GH, but looks like code wasn't updated.

Reply:
@wa0x6e in case you missed this comment.

Reply:
Fixed, forgot to push 🙈